source/fundamentals/connection/connect.txt

```diff
@@ -17,8 +17,8 @@ Connection Guide
    :depth: 2
    :class: singlecol
 
-This guide shows you how to connect to a MongoDB instance or replica set
-using the Node.js driver.
+This guide shows you how to connect to a MongoDB Atlas instance, 
```

Comment:
Suggestion:

I think the terminology Atlas uses is "cluster" (see https://www.mongodb.com/docs/atlas/manage-clusters/)

```suggestion
This guide shows you how to connect to a MongoDB Atlas cluster, 
```

Reply:
> Suggestion:
> 
> I think the terminology Atlas uses is "cluster" (see https://www.mongodb.com/docs/atlas/manage-clusters/)

The term "cluster" actually doesn't include serverless instances, which can also connect via Node driver. If we want to use a term other than "instance," we could use "database deployment" or "deployment" as those are usually used in the Atlas docs to encompass both clusters and serverless

Reply:
I'll go with "deployment" here to encompass both cases. Thank you both for the suggestions/info!